contrib/archlinux/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+pkgname=fido2-hmac-secret-git
+_pkgname=fido2-hmac-secret
+pkgver=0.3.0.r12.ge474de5
+pkgrel=1
+pkgdesc="A simple way to generate password-proteceted secrets from a FIDO2 authenticator with the hmac-secret extension"
+arch=('x86_64')
+url="https://github.com/mjec/fido2-hmac-secret"
+license=('GPL3')
+depends=('bash' 'libfido2' 'libcbor' 'libsodium' 'libcap')
+makedepends=('clang' 'git')
+source=("git+$url.git")
+sha256sums=('SKIP')
+install="$_pkgname.install"
+
+pkgver() {
+  cd "$_pkgname"
+  git describe --long --tags | sed 's/^v//;s/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+build() {
+  cd "$_pkgname"
+  export CC="gcc"
+  make release bash-completion initcpio
```

Comment:
I'm not a fan of that because there is no guarantee that environment where package is being build is the same as those where it will be installed i.e. building in clean chroot (which is mandatory if this will be ever put in official repos).

Moreover it's possible that user will install `bash-completion` after they build this one and in that case they would have to rebuild this package again to get functionality.

To sum up I think that build process should be deterministic.

Reply:
Alright I buy that